src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(double firstLeg, double secondLeg) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return firstLeg * secondLeg / 2;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: " + this.getClass().getSimpleName() + ", area: "
+                + getArea() + " sq.units, firstLeg: " + firstLeg
+                + " units, secondLeg: " + secondLeg + " units, color: " + getRandomColor();
```

Comment:
this figure doesn't have color, it just generates random one for console print. Create field according to task description